lib/workers/global/config/parse/__fixtures__/config-async-function.js

```diff
@@ -0,0 +1,8 @@
+// This is functionally equivalent to config-function-promise.js but syntactically different
```

Comment:
I suspect we should "link" this PR to issue #13035, so that it's closed when the PR gets merged. Right now this PR doesn't formally close any issue.